### ISSUE TITLE
Restrict installation of bokeh to 2.4.

### ISSUE THREAD

**Issue by sybenzvi** [desihub/desiconda#64]
Conda now installs bokeh 3.1 by default; this is not compatible with software such as Nightwatch. We may want to specify

`
bokeh=2.4
`

in the requirements.

**Comment by weaverba137** [comment]
`conf/conda-pkgs.sh` has `"bokeh<3"`, so I don't think this is an issue any longer. @sybenzvi, do you agree?

**Comment by sbailey** [comment]
I agree the "bokeh<3" constraint addresses the underlying concern.  Closing this ticket.  Separately we'll need to eventually upgrade nightwatch, prospect, etc. to bokeh3.

**Comment by weaverba137** [comment]
@sbailey, We already upgraded prospect to support Bokeh 3: desihub/prospect#111.
